Userland/Applications/Assistant/Providers.cpp

```diff
@@ -135,7 +135,7 @@ void FileProvider::query(DeprecatedString const& query, Function<void(Vector<Non
             Vector<NonnullRefPtr<Result>> results;
 
             for (auto& path : m_full_path_cache) {
-                if (task.is_cancelled())
+                if (task.is_canceled())
```

Comment:
`cancelled` was nice for me. 😅
(not sure what the preferred language dialect is, but as far as i remember we go more for the europe? 🤔 — it seems youʼre using `cancelled` in other places)

Reply:
https://github.com/SerenityOS/serenity/blob/master/CONTRIBUTING.md#human-language-policy

Reply:
so thereʼs `cancelled` somewhere below then! 😁